docs/category-overview-pages/accessing-netdata-dashboards.md

```diff
@@ -1,3 +1,28 @@
 # Accessing Netdata Dashboards
 
-This section contains documentation on how you can access the Netdata Agent's dashboards, and the Netdata Cloud's dashboards.
\ No newline at end of file
+This section contains documentation on how you can access the Netdata dashboards, which are the same both for the Agent and Cloud.
+
+A user accessing the Netdata dashboard **from the Cloud** will always be presented with the latest Netdata dashboard version.
+
+A user accessing the Netdata dashboard **from the Agent** will, by default, be presented with the latest Netdata dashboard version (which is the one also used on Netdata Cloud).
+The exceptions happen when the:
+* Agent doesn't have external network access, so it won't be able to get the latest Netdata dashboards, but will fallback to the Netdata dashboard version that 
+was shipped with the agent;
+* Users have defined, e.g. through URL bookmark, that they wants to see the previous version of the dashboard (accessible `http://NODE:19999/v1`, replacing `NODE` with the IP address or hostname of your Agent). 
+
+## Main sections
+
+The Netdata dashboard consists of the following main sections:
+* [Netdata charts](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/interact-new-charts.md)
+* [Infrastructure Overview](https://github.com/netdata/netdata/blob/master/docs/visualize/overview-infrastructure.md)
+* [Nodes view](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/nodes.md)
+* [Custom dashboards](https://learn.netdata.cloud/docs/visualizations/custom-dashboards)
+* [Alerts](https://github.com/netdata/netdata/blob/master/docs/monitor/view-active-alarms.md)
+* [Anomaly Advisor](https://github.com/netdata/netdata/blob/master/docs/cloud/insights/anomaly-advisor.md)
+* [Functions](https://github.com/netdata/netdata/blob/master/docs/cloud/netdata-functions.md)
+* [Events feed](https://github.com/netdata/netdata/blob/master/docs/cloud/insights/events-feed.md)
+
+> ⚠️ Some sections will have restricted functionality but, where possible, you will still be able to fully experience the features having the dependency on a _last step_ for you to be signed-in to Netdata Cloud, e.g saving visualization settings on charts or custom dashboards, or claiming the node to Netdata Cloud, e.g. executing function on an Agent.
```

Comment:
I might write it as:

```
Some sections of the dashboard may require the user to be signed-in to Netdata Cloud for their full functionality. Examples include saving visualization settings on charts or custom dashboards, claiming the node to Netdata Cloud, or executing functions on an Agent.
```

Reply:
I like your suggestion @MrZammler and maybe it is more proper since, for example, functions users won't be able to experience it without a Cloud account and having the node claimed
will update it 👍